ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-workflow.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-workflow.yaml",
```

Comment:
To keep consistent with the existing workflow naming, suggest to rename `sts` to `cco-manual-security-token-service`